pepper/cli.py

```diff
@@ -377,6 +386,7 @@ def parse_cmd(self):
             low['fun'] = args.pop(0)
             low['batch'] = self.options.batch
             low['arg'] = args
+            low['full_return'] = True
```

Comment:
So this will change the response shape from salt-api, is the CLI output changing too or does that stay the same? If that changes you might want to flag this or cut a major release.

Reply:
Initially i did not notice a difference, but i think there might be.  I will double check that

Reply:
i verified, and it still prints out in the same format.